nuxeo-drive-core/src/main/java/org/nuxeo/drive/adapter/impl/DocumentBackedFolderItem.java

```diff
@@ -456,8 +457,11 @@ public FolderItem createFolder(String name, boolean overwrite) {
     public FileItem createFile(Blob blob, boolean overwrite) {
         String fileName = blob.getFilename();
         try (CloseableCoreSession session = CoreInstance.openCoreSession(repositoryName, principal)) {
-            DocumentModel file = getFileManager().createDocumentFromBlob(session, blob, docPath, overwrite, fileName,
-                    false, true);
+            FileImporterContext context = FileImporterContext.builder(session, blob, docPath)
+                                                             .overwrite(overwrite)
+                                                             .excludeOneToMany(true)
+                                                             .build();
```

Comment:
What about the `filename`?

Reply:
No need to set it as by default we are using the Blob's filename.